assets/js/Module.js

```diff
@@ -41,7 +42,11 @@ function collect(connect, monitor) {
 class Module extends React.Component {
   constructor(props) {
     super(props);
-    this.itemClick = this.itemClick.bind(this);
+    this.itemClick = debounce(this.itemClick.bind(this), 1000, {
```

Comment:
Logic looks fine. I’d like the comment to  why we are doing this  (Suppress multiple adds and dbl click)